labs-ios-starter/Map/LocationSearchTableViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  LocationSearchTableViewController.swift
+//  labs-ios-starter
+//
+//  Created by Claudia Maciel on 1/20/21.
+//  Copyright © 2021 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+
+class LocationSearchTableViewController: UITableViewController {
+
+    var matchingItems:[MKMapItem] = []
+    var mapView: MKMapView? = nil
+    
+    var handleMapSearchDelegate: HandleMapSearch? = nil
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+    }
```

Comment:
If you aren't planning on doing anything to override the default behavior of `viewDidLoad` for this view controller, you can just remove all of this